backend/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Base image for the container
-FROM node:20.4.0
+FROM node:20.14.0
```

Comment:
Did it break something  with 20.4? Would like me to investigate some other smaller images to try and save some resources?

Reply:
Its just keeping up to date. If nothing breaks, always step up.

Reply:
BTW smaller images are a great idea!  We have "ways" of chopping down too, but that falls under optimization.